worldwind/src/main/java/gov/nasa/worldwind/shape/Label.java

```diff
@@ -485,7 +485,13 @@ protected void makeDrawable(RenderContext rc) {
         // Configure the drawable according to the active attributes. Use a color appropriate for the pick mode. When
         // picking use a unique color associated with the picked object ID. Use the texture associated with the active
         // attributes' text image and its associated tex coord transform.
-        drawable.color.set(rc.pickMode ? renderData.pickColor : this.activeAttributes.textColor);
+        if (rc.pickMode) {
+            drawable.color.set(renderData.pickColor);
+        } else {
+            // The text texture includes the appropriate color for drawing, specifying white here ensures the color
+            // multiplication in the shader results in the texture's color.
+            drawable.color.set(new Color(1f, 1f, 1f, 1f));
```

Comment:
There's a per-frame allocation here. We don't do this in Android. Why use `set(Color)` with a constant when there's `set(float, float, float, float)`?

Could you also incorporate your comment in the block above?